### ISSUE TITLE
You'll Need a Bear (ID= 12972)

### ISSUE THREAD

**Issue by mmamani93** [dalaranwow/dalaran-wow#5279]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: NPCs from Valey of Ancient Winters (Brijana and Warebears) vanish as soon as I enter into Valey of Ancient Winters (where they are supposed to be). But they are visible if I'm in Brunnhildar Village.
I have tried with a different character that hasn't done the quest chain, and it was able to see NPCs and interact with them.

**Expected behaviour**: NPCs should be there when you go into Valey of Ancient Winters so you can deliver quest and use the Warebears to complete it.

**Steps to reproduce the problem**:

1.  Complete quest chain until You'll Need a Bear (ID= 12972)
2. Try to deliver the quest in the marked point in the map.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
NPCs being visible from Brunnhildar Village.
![foto 1](https://user-images.githubusercontent.com/19556067/35568860-23ff1066-05a9-11e8-8368-1abab6bed452.PNG)
NPCs vanished being in Valey of Ancient Winters
![foto 2](https://user-images.githubusercontent.com/19556067/35568864-263a8040-05a9-11e8-8f91-ed6b90214346.PNG)

**Comment by Lothloryen** [comment]
Hello there,

Your are getting phased because you have the quest [Forging an Alliance](http://www.wowhead.com/quest=12924/forging-an-alliance), remove it and you should be fine.

**Comment by mmamani93** [comment]
Wow, never will release of that. Thank you so much
